src/libs/actions/Link.js

```diff
@@ -35,28 +35,33 @@ function showGrowlIfOffline() {
  * @param {String} url
  */
 function openOldDotLink(url) {
-    if (showGrowlIfOffline()) {
-        return;
-    }
-
+    /**
```

Comment:
I feel that there should be an extra line space above, but might be wrong about it.

Reply:
Eslint formats it to be without the extra line